stile/sys_tests.py

```diff
@@ -891,13 +891,392 @@ def __call__(self, array, linewidth = 0.01, scale = None, figsize = None,
+                  histtype='step', xlabel=None, ylabel=None,
+                  xlim=None, ylim=None, hide_x=False, hide_y=False,
+                  cumulative=False, align='mid', rwidth=0.9,
+                  log=False, color='k', alpha=1.0,
+                  text=None, text_x=0.90, text_y=0.90, fontsize=12,
+                  linewidth=2.0, vlines=None, vcolor='k' ):
+
+        """
+        Draw a histogram and return a `matplotlib.figure.Figure` object.
+
+        This method has a bunch of options for controlling the appearance of
+        the histogram, which are explained below.
+
+        To implement a child class of HistogramSysTest, call HistoPlot within
+        __call__ of the child class and return the `matplotlib.figure.Figure`
+        that HistoPlot returns.
+
+        @param data_list     The 1-Dimension NumPy array or a list of Numpy arrays
+                             for plotting histograms.
+
+        @param binning_style Different selections of Histogram bin size:
+               = 'scott' :   Using the Scott's rule to decide the bin size.
+               = 'freedman': Using the Freedman-Diaconis rule to decide the bin
+                             size.
+               = 'manual' :  Manually select a fixed number of bins.
+                             [default: binning_style='manual']
+
+        @param nbins         The number of bins if binning_style = 3 is selected.
+                             [Default: nbins = 50]
+        @param weights       An array of weights.
+        @param limits        The [min, max] limits to trim the data before the
+                             histogram is made.
+                             [Default: limits = None]
+        @param normed        Whether the normalized histogram is shown.
+                             [Default: normed = False]
+        @param cumulative    Whether the cumulative histogram is shown.
+                             [Default: cumulative = False]
+        @param histtype      The type of histogram to show:
+               histtype = 'bar'        : Tradition bar-type histogram.
+               histtype = 'step'       : Unfilled lineplot-type histogram.
+               histtype = 'stepfilled' : Filled lineplot-type histogram.
+                             [Default: histtype = 'stepfilled']
+        @param align         Where the bars are centered relative to bin edges
+                             = 'left', 'mid', or 'right'.
+                             [Default: align = 'mid' ]
+        @param rwidth        The relative width of the bars as a fraction of the
+                             bin width. Ignored for histtype = 'step' or
+                             'stepfilled'.
+                             [Default = None]
+        @param log           If True, the histogram axis will be set to a log scale.
+                             [Default = False]
+        @param color         Color of the histogram.
+                             [Default = None]
```

Comment:
What does Default = None for color mean in this case?  (matplotlib defaults, I assume?)